test/test.js

```diff
@@ -25,24 +25,6 @@ describe('nodejs-docker', () => {
       }, 3000);
     });
   });
-
-  it('can use a custom nodejs version', (done) => {
```

Comment:
This test used to get run on travis before, but now it doesn't. Conversely, the other tests in this file don't get run as part of the structure_tests. Is there a way to reconcile these?